packages/svelte-check/src/index.ts

```diff
@@ -30,11 +30,43 @@ async function openAllDocuments(
     filePathsToIgnore: string[],
     svelteCheck: SvelteCheck
 ) {
-    const files = await glob('**/*.svelte', {
-        cwd: workspaceUri.fsPath,
-        ignore: ['node_modules/**'].concat(filePathsToIgnore.map((ignore) => `${ignore}/**`))
+    const offset = workspaceUri.fsPath.length + 1;
+    // We support a very limited subset of glob patterns: You can have a * at the end or the start
+    const ignored: Array<(path: string) => boolean> = filePathsToIgnore.map((i) => {
+        if (i.endsWith('*')) i = i.slice(0, -1);
```

Comment:
I think this should be `**`. because the `path.includes` call below won't care about path separators

Reply:
what do you mean by that? Can you give an example where it would break / be wrong?

Reply:
`*` vs `**` have different meaning in glob syntax where `*` excludes path separators and `**` includes them. See https://github.com/micromatch/picomatch?tab=readme-ov-file#basic-globbing

`path.includes` will not stop at path separators like `*` will

So if you want to ignore all of `node_modules` the glob syntax `node_modules/*` should not work whereas `node_modules/**` should

Reply:
ah ok so this is a case of me not knowing the globbing syntax well enough 😄